data/hails.txt

```diff
@@ -662,6 +678,7 @@ phrase "friendly civilian"
 		"Finally dropped off some"
 		"Dropped off my cargo of"
 		"Just made a big delivery of"
+		"Recently got rid of"
```

Comment:
Have you tried saying these out loud? The first one feels like the verbal equivalent of running into a brick wall. OK the second one is not "correct" but at the very least it doesn't feel awkward.